CveXplore/core/database_maintenance/api_handlers.py

```diff
@@ -34,9 +35,16 @@ def process_item(self, item: dict):
                 # ).entry
                 return item
 
+    @staticmethod
+    def split_cpe_name(cpename: str):
+        """Split CPE name into its components, accounting for escaped colons."""
+        non_escaped_colon = r"(?<!\\):"
+        split_name = re.split(non_escaped_colon, cpename)
+        return split_name
```

Comment:
This function is identical to the one in `common/cpe_converters.py`, isn't it?
Could it be used from there instead of adding a copy?

Reply:
fixed by https://github.com/cve-search/CveXplore/pull/269/commits/e433be98aa21874228e1ba7a77fc92e4205ab2ec